music21/musicxml/m21ToXml.py

```diff
@@ -5523,7 +5522,17 @@ def articulationToXmlTechnical(self, articulationMark):
             mxTechnicalMark.text = str(articulationMark.number)
         if musicXMLTechnicalName == 'fret':
             mxTechnicalMark.text = str(articulationMark.number)
-
+        if musicXMLTechnicalName == 'bend':
+            bendAlterSubElement = SubElement(mxTechnicalMark, 'bend-alter')
+            bendAlterSubElement.text = str(articulationMark.bendAlter.semitones)
```

Comment:
this has enough ifs that it should be brought out as its own small method that allows for easier documentation and testing.  articulationMark will probably need to be cast as a FretBend.

We can solve the case of `GenericInterval` later, which doesn't have .semitones.  Perhaps it always needs to be a full Interval object or None.

Reply:
New submethod added in https://github.com/cuthbertLab/music21/pull/1580/commits/85cc6a8cea3540b6397e309a5b377d45eb0c2d0c
Now I indeed have the problem that I need to retrieve the number of semitones. I could probably do something with `GenericInterval.value` but I don't know how to do the conversion properly